src/Organizations/OrganizationForm/OrganizationSummaryForm/FieldOrganizationTypes/FieldOrganizationTypes.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+import {
+    find,
+    map,
+  } from 'lodash';
+
+import { FieldMultiSelectionFinal } from '@folio/stripes-acq-components';
+
+export const FieldOrganizationTypes = ({ organizationTypes, mutators }) => {
+  const formatter = ({ option }) => {
+    const item = find(organizationTypes, { id: option }) || option;
+
+    if (!item) return option;
+
+    return item.name;
+  };
+
+  const itemToString = item => item;
+
+  const typeOptions = map(organizationTypes, 'id');
+
+  const filter = (filterText, list) => {
+    const renderedItems = filterText
+      ? organizationTypes
+        .filter(group => group.name.toLowerCase().includes(filterText.toLowerCase()))
+        .map(({ id }) => id)
+      : list;
+
+    return { renderedItems };
+  };
+
+  const getSelectedTypes = (e) => {
```

Comment:
I believe you don't have to use mutators to change form value in this case. `onChange` of `FieldMultiSelectionFinal` provide callback with selected values. You can use `change` to update the `organizationTypes` value. 

```suggestion
  const onChangeTypes = (types) => {
    change('organizationTypes', types);
  };
```

Reply:
Look how it works in `OrganizationVendorInfoForm` component.

Reply:
Thanks for the hint!